dbcon/joblist/rowestimator.cpp

```diff
@@ -199,7 +200,7 @@ uint32_t RowEstimator::estimateDistinctValues(const execplan::CalpontSystemCatal
         ret = max - min + 1;
     }
 
-    if (ret > fRowsPerExtent)
+    if (ret > (T) fRowsPerExtent)
```

Comment:
Could you ellaborate on this cast?

Reply:
I agree, this cast is not required. I am updating this.